include/pmacc/memory/boxes/DataBoxDim1Access.hpp

```diff
@@ -48,16 +48,26 @@ namespace pmacc
         {
         }
 
-        HDINLINE DataBoxDim1Access(DataBoxDim1Access const&) = default;
+        DataBoxDim1Access(DataBoxDim1Access const&) = default;
```

Comment:
Just a doubt. I see in some other places in the code that copy constructors are marked as HDINLINE. I'm not sure I understand the implications of this removal.

Reply:
for default constructors, the host/device attribute is not required. It is a left over from old times.